session/bench_test.go

```diff
@@ -55,35 +55,35 @@ func prepareBenchSession() (Session, *domain.Domain, kv.Storage) {
 
 func prepareBenchData(se Session, colType string, valueFormat string, valueCount int) {
 	mustExecute(se, "drop table if exists t")
-	mustExecute(se, fmt.Sprintf("create table t (pk int primary key auto_increment, col %s, index idx (col))", colType))
+	mustExecute(se, "create table t (pk int primary key auto_increment, col %n, index idx (col))", colType)
 	mustExecute(se, "begin")
 	for i := 0; i < valueCount; i++ {
-		mustExecute(se, "insert t (col) values ("+fmt.Sprintf(valueFormat, i)+")")
+		mustExecute(se, "insert t (col) values (%?)", i)
```

Comment:
valueFormat may not be dropped easily, check that.

Reply:
There is one caller to `prepareSortBenchData`, and it uses the format '%v', so that is straight forward in that it can be eliminated.

For `prepareBenchData`, the value format doesn't really make sense since it is always on an integer `i`. It is always %v, %d, or %v.1234. Since the %v.1234 is on a precision type (decimal), I don't quite understand how it is expected to work. It will just have a small change in parsing performance? But this is the prepare phase of the bench, the values will remain the same.

So I'll remove both `valueFormat` parameters. It might be that they were intended to work differently at some point, but survived a refactor. I'll clean it up for the next person.

Reply:
Oops, I though you have removed the other two. Whatever, it makes sense to me. I think he just wants to generate some floats by decimals.

Reply:
Actually, it looks like I misread it. I'm going to restore the valueFormat for `prepareBenchData` and wrap it in a sprintf.